diff --git a/node_modules/@mdxeditor/editor/dist/plugins/core/index.js b/node_modules/@mdxeditor/editor/dist/plugins/core/index.js
index 8a897ec..a3879a5 100644
--- a/node_modules/@mdxeditor/editor/dist/plugins/core/index.js
+++ b/node_modules/@mdxeditor/editor/dist/plugins/core/index.js
@@ -284,7 +284,7 @@ const createRootEditorSubscription$ = Appender(rootEditorSubscriptions$, (r, sig
             jsxIsAvailable: r.getValue(jsxIsAvailable$)
           });
         });
-        r.pub(markdown$, theNewMarkdownValue.trim());
+        r.pub(markdown$, theNewMarkdownValue.replace(/\n*$/, "\n"));
         r.pub(initialMarkdownNormalize$, false);
       });
     },
